stm32cube/stm32u3xx/README

```diff
@@ -0,0 +1,51 @@
+STM32CubeU5
+###########
+
+Origin:
+   ST Microelectronics
+   https://www.st.com/en/embedded-software/stm32cubeu3.html
+
+Status:
+   version v1.2.0
```

Comment:
Only V1.1.0  is available here: https://github.com/STMicroelectronics/STM32CubeU3.